README.md

```diff
@@ -1,4 +1,4 @@
-# gtable
+# gtable <img src="man/figures/logo.png" align="right" height=140/>
```

Comment:
The logo should be 120 x 139, and you shouldn't need to specify height here (just to be consistent with the other repos)

Reply:
hmm... we need to update usethis then, as this is the output it gives